gaurun/apns_http2.go

```diff
@@ -54,6 +58,34 @@ func NewApnsClientHttp2(certPath, keyPath string) (*http.Client, error) {
 	}, nil
 }
 
+func loadX509KeyPairWithPassword(certPath, keyPath, keyPassphrase string) (tls.Certificate, error) {
+	keyPEMBlock, err := ioutil.ReadFile(keyPath)
+	if err != nil {
+		return tls.Certificate{}, err
+	}
+	if keyPassphrase != "" {
+		pemBlock, _ := pem.Decode(keyPEMBlock)
+		if !x509.IsEncryptedPEMBlock(pemBlock) {
+			err = errors.New(keyPath + "is not encrypted. passphrase is not required")
```

Comment:
Why don't you use `fmt.Errorf()` instead of `errors.New()`? When variable is used, it's more readable.